subsys/shell/modules/kernel_service.c

```diff
@@ -44,13 +44,45 @@ static int cmd_kernel_version(const struct shell *sh,
 	return 0;
 }
 
-static int cmd_kernel_uptime(const struct shell *sh,
-			     size_t argc, char **argv)
+#define MINUTES_FACTOR (MSEC_PER_SEC * 60)
+#define HOURS_FACTOR   (MINUTES_FACTOR * 60)
+#define DAYS_FACTOR    (HOURS_FACTOR * 24)
```

Comment:
Would it make sense to add new definitions for these in `include/zephyr/sys_clock.h`? E.g. `SEC_PER_MIN`, `MIN_PER_HOUR`, ...

Reply:
sure